server/routes/visit.test.ts

```diff
@@ -254,73 +254,82 @@ describe('/visit/:reference', () => {
         })
     })
 
-    it('should render full booking summary page with visit information and prisoner tab selected with search back link when from visits', () => {
-      const url =
-        '/visit/ab-cd-ef-gh?query=startDate%3D2022-05-24%26type%3DOPEN%26time%3D3pm%2Bto%2B3%253A59pm&from=visit-search'
+    describe('back links', () => {
```

Comment:
Tests added for all the possible 'Back' link permutations on the booking summary page.